zebra/zebra_vty.c

```diff
@@ -455,6 +455,10 @@ static void vty_show_ip_route(struct vty *vty, struct route_node *rn,
 				    re->status);
 		json_object_int_add(json_route, "internalFlags",
 				    re->flags);
+		json_object_int_add(json_route, "internalnhnum",
```

Comment:
I just suggest picking the more distinguished names like `internalNextHopNum` and `internalNextHopActiveNum`

Reply:
To jump on here, we are using camel case for json output so please follow that presentation too.